### ISSUE TITLE
Plugin attempts to set volume when AVR is off

### ISSUE THREAD

**Issue by 11harveyj** [crowbarz/ha-pioneer_async#10]
Seem to be seeing a few of these errors in my HA logs at the moment:

```
Logger: aiopioneer.pioneer_avr
Source: custom_components/pioneer_async/media_player.py:412
Integration: Pioneer AVR ([documentation](https://github.com/crowbarz/ha-pioneer_async))
First occurred: 16 August 2022 at 22:32:33 (8 occurrences)
Last logged: 21 August 2022 at 20:37:53

AVR command 073VL returned error: E02
```

It appears that this happens when the AVR is switched off which according to the docs that I have:

E02<CR+LF> | NOT AVAILABLE   NOW | Detected   the Command line which could not work now. |  
-- | -- | -- | --

Perhaps a check needs to be added to ensure that the AVR is powered on?

**Comment by crowbarz** [comment]
Yes, this is expected behaviour, though the HA UI guards against being able to send volume up/down/set service calls to media players when they are off. HA does not however prevent service calls from being directly made to a media player that is off, for any media player. So it would be useful to understand what else was happening on your system (eg. scripts, automations, external events) immediately before the volume set command.

As far as I can see in the code, volume set is only ever sent to the AVR if requested via a service call.

**Comment by 11harveyj** [comment]
My apologies, was tired when I posted this... indeed it is an automation lol.

Thank you for having a look :)
